src/Assimp/Silk.NET.Assimp/Structs/Scene.gen.cs

```diff
@@ -219,16 +207,6 @@ public Scene
         [NativeName("Name", "mName")]
         public AssimpString MName;
 
-        [NativeName("Type", "unsigned int")]
-        [NativeName("Type.Name", "unsigned int")]
-        [NativeName("Name", "mNumSkeletons")]
-        public uint MNumSkeletons;
-
-        [NativeName("Type", "aiSkeleton **")]
-        [NativeName("Type.Name", "aiSkeleton **")]
-        [NativeName("Name", "mSkeletons")]
-        public Skeleton** MSkeletons;
-
```

Comment:
Bug or upstream change?

Reply:
Perhaps an un-updated submodule? ill try updating the assimp submodule (why does every single-binding update PR end up touching everything? am i cursed?)

Reply:
yup was just old submodule, fixed